test_scripts/API/VehicleData/commonVehicleData.lua

```diff
@@ -13,6 +13,9 @@ config.defaultProtocolVersion = 2
 
 --[[ Local Variables ]]
 local m = {}
+m.cprint = utils.cprint
+m.getPreloadedPT = actions.sdl.getPreloadedPT
+m.setPreloadedPT = actions.sdl.setPreloadedPT
```

Comment:
@GetmanetsIrina I would suggest to move these functions to `Shared Functions` section

Reply:
@dboltovskyi, updated in f4c313695cf4446ed5a2e3d876d26609f2b49de9